packages/host/tests/acceptance/interact-submode-test.gts

```diff
@@ -1491,12 +1490,11 @@ module('Acceptance | interact submode tests', function (hooks) {
       await click(`[data-test-search-result="${testRealmURL}Person/fadhlan"]`);
       // There is some additional thing we are waiting on here, probably the
       // card to load in the card resource, but I'm not too sure so using waitUntil instead
-      await waitUntil(() =>
-        document
-          .querySelector(
-            '[data-test-operator-mode-stack="1"] [data-test-stack-card-index="0"]',
-          )
-          ?.textContent?.includes('Fadhlan'),
+      await waitUntil(
```

Comment:
In case you gained knowledge about what exactly we need to waitUntil for here, you can adjust the comment above